packages/gen-ai/frontend/src/app/Chatbot/components/alerts/SourceUploadErrorAlert.tsx

```diff
@@ -6,17 +6,22 @@ interface SourceUploadErrorAlertProps {
   isVisible: boolean;
   alertKey: number;
   onClose: () => void;
+  errorMessage?: string;
 }
 
 const SourceUploadErrorAlert: React.FunctionComponent<SourceUploadErrorAlertProps> = ({
   isVisible,
   alertKey,
   onClose,
+  errorMessage,
```

Comment:
Could be simplified as

```suggestion
  errorMessage = 'Please try again.',
```
then remove `defaultMessage` and `displayMessage` below